src/integrationTest/java/uk/gov/hmcts/reform/iataskconfiguration/CamundaCancellationTaskTest.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.hmcts.reform.iataskconfiguration;
+
+import org.camunda.bpm.dmn.engine.DmnDecision;
+import org.camunda.bpm.dmn.engine.DmnDecisionTableResult;
+import org.camunda.bpm.dmn.engine.DmnEngine;
+import org.camunda.bpm.dmn.engine.DmnEngineConfiguration;
+import org.camunda.bpm.engine.variable.VariableMap;
+import org.camunda.bpm.engine.variable.impl.VariableMapImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+class CamundaCancellationTaskTest {
+
+    private DmnEngine dmnEngine;
+
+    @BeforeEach
+    void setUp() {
+        dmnEngine = DmnEngineConfiguration
+            .createDefaultDmnEngineConfiguration()
+            .buildEngine();
+    }
+
+
+    @DisplayName("Set auto cancellation")
+    @Test
+    void set_auto_cancellation() {
```

Comment:
suggestion: you can parametrise this test and cover both inputs for the DMN table using the same test.